investment_tracking/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from startups.models import StartUpProfile
+from investors.models import InvestorProfile
+
+
+class InvestmentTracking(models.Model):
+    """
+    Model for tracking investments made by investors in startups.
+
+    Attributes:
+        investor_id (ForeignKey): The investor associated with the investment.
+        startup_id (ForeignKey): The startup associated with the investment.
+        saved_at (DateTimeField): The timestamp when the investment was recorded.
+
+    Methods:
+        __str__(): Returns a string representation of the investment tracking entry.
+    """
+     
+    investor_id = models.ForeignKey(InvestorProfile, on_delete = models.CASCADE)
+    startup_id = models.ForeignKey(StartUpProfile, on_delete = models.CASCADE)
```

Comment:
Using `investor` and `startup` as field names instead of `investor_id` and `startup_id` is more natural in Django. By default, Django will add `_id` when generating database column names, so you don’t need to specify it in the model.